src/Sylius/Behat/Context/Api/Shop/CheckoutContext.php

```diff
@@ -127,6 +148,17 @@ public function iAmAtTheCheckoutAddressingStep(): void
         // Intentionally left blank
     }
 
+    /**
+     * @When /^I choose "([^"]+)" street for (billing|shipping) address$/
+     */
+    public function iChooseForBillingAddress(string $street, string $addressType): void
+    {
+        $this->sharedStorage->set(
+            'filled_address_field',
+            ['street' => $street, 'addressType' => $addressType . 'Address']
+        );
```

Comment:
I guess you should here get an address from address book and fill it in content data